database/seeds/UserAdditionsSeeder.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use App\Models\Users\User;
+use App\Models\Users\Profile;
+use Illuminate\Database\Seeder;
+use App\Models\Users\UserNotification;
+
+class UserAdditionsSeeder extends Seeder
```

Comment:
Not sure why the dev/master version is really old, but https://github.com/MultiCrew/copilot/blob/feature/profiles/database/seeds/UserSeeder.php has a method which is called each time to add a user and does all of the stuff below. Can you refactor this so that any steps carried out by this seeder are just part of the method in the main UserSeeder class?

Reply:
No as this is meant to be run by itself on prod to insert the required models to already existing users rather than make them have to create new accounts or do it manually